src/Sandbox/FPSCounter.cpp

```diff
@@ -0,0 +1,58 @@
+#include "FPSCounter.h"
+
+#include <string>
+#include <GLFW/glfw3.h>
+
+#include "Utils/EngineExceptions.h"
+
+FPSCounter::FPSCounter()
+{
+    if (!gltInit())
+	{
+		throw BadInitializationException("Failed to initialize GLText library!");
+	}
+
+    this->_lastTime = glfwGetTime();
+    this->_numberOfFrames = 0;
+    this->_actualFps = 60;
+    this->_fps = gltCreateText();
+
+}
+
+
+void FPSCounter:: CountFps() {
```

Comment:
Please use the [Allman convention](https://en.wikipedia.org/wiki/Indentation_style#:~:text=somethingelse()%3B%0A%7D-,Allman,-while%20(x).

Reply:
Also please justify the indentation